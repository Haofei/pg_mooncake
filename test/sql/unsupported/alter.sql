CREATE TABLE t (a int) USING columnstore;
ALTER TABLE t ADD COLUMN b int;
DROP TABLE t;
