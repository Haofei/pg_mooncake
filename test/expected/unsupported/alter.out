CREATE TABLE t (a int) USING columnstore;
ALTER TABLE t ADD COLUMN b int;
ERROR:  ALTER TABLE on columnstore table is not supported
DROP TABLE t;
